src/Entry.hs

```diff
@@ -20,8 +20,8 @@ data Entry = Entry
 
 
 showEntry :: Entry -> String
-showEntry (Entry index date message) =
-    prettyIndex index ++ "  " ++ date ++ "  " ++ message
+showEntry (Entry i d m) =
+    prettyIndex i ++ "  " ++ d ++ "  " ++ m
```

Comment:
not sure what best practice is here ... I ran `stack build` with the `--pedantic` flag, and this was saying that these were already defined up above, but I thought they'd be okay in this scope.